### ISSUE TITLE
2.0 tarball missing "configure" script (INSTALL says it should be there)

### ISSUE THREAD

**Issue by chuckcranor** [CCI/cci#43]
% ls
v2.0.tar.gz
% tar xzf v2.0.tar.gz 
% cd cci-2.0/
cwd: /tmp/c/cci-2.0
% ls configure
ls: configure: No such file or directory
% head INSTALL
= Common Communication Interface Installation Guide ============================

From a tarball (.tar.gz)

To install CCI from a tarball, simply execute the following commands, where
<version> is the CCI version you want to install:

        tar xzf cci-<version>.tar.gz
        cd cci-<version>
        ./configure
% ./autogen.pl 
CCI autogen
1. Searching for projects, frameworks, and plugins...

This doesn't look like a developer copy of CCI.  You probably do not
want to run autogen.pl - it is normally not needed for a release
source tree.  Giving you 5 seconds to reconsider and kill me.

^C
%

**Comment by scottatchley** [comment]
Hi,

Thanks for the heads up. I will post a new tarball this weekend.

Scott

**Comment by scottatchley** [comment]
I just downloaded and checked the tarball:

% tar tzf cci-2.0.tar.gz 
cci-2.0/
cci-2.0/aclocal.m4
cci-2.0/autogen.pl
cci-2.0/cci.pc.in
cci-2.0/config/
cci-2.0/config/ar-lib
cci-2.0/config/autogen_found_items.m4
cci-2.0/config/cci.m4
cci-2.0/config/cci_check_attributes.m4
cci-2.0/config/cci_check_package.m4
cci-2.0/config/cci_check_vendor.m4
cci-2.0/config/cci_check_visibility.m4
cci-2.0/config/cci_check_withdir.m4
cci-2.0/config/cci_functions.m4
cci-2.0/config/cci_get_version.sh
cci-2.0/config/cci_plugins.m4
cci-2.0/config/cci_setup_plugin_package.m4
cci-2.0/config/compile
cci-2.0/config/config.guess
cci-2.0/config/config.sub
cci-2.0/config/depcomp
cci-2.0/config/distscript.csh
cci-2.0/config/install-sh
cci-2.0/config/libtool.m4
cci-2.0/config/ltmain.sh
cci-2.0/config/ltoptions.m4
cci-2.0/config/ltsugar.m4
cci-2.0/config/ltversion.m4
cci-2.0/config/lt~obsolete.m4
cci-2.0/config/missing
cci-2.0/config/test-driver
cci-2.0/configure
…

I checked the configure script and it seems fine. Can you download and try again?

Scott

**Comment by bgoglin** [comment]
Given the name v2.0.tar.gz, I think he took the tarball generated by github when you push a tag. They are available from https://github.com/CCI/cci/releases, but it is just a tarball of the git tree, without running autogen. Unfortunately, there's currently now way to disable these automatic and incomplete tarballs as far as I know.

**Comment by chuckcranor** [comment]
Yes, I got it from here: https://github.com/CCI/cci/archive/v2.0.tar.gz.  I was on the cci github page https://github.com/CCI/cci and clicked on "releases" ... seemed like a logical place to look.
